trappy/ftrace.py

```diff
@@ -215,7 +215,8 @@ def __populate_data(self, fin, cls_for_unique_word, window, abs_window):
                 return
 
             # Remove empty arrays from the trace
-            data_str = re.sub(r"[A-Za-z0-9_]+=\{\} ", r"", data_str)
+            if "={}" in data_str:
```

Comment:
Super cool :heart_eyes: 

I would just suggest to strip down the commit title to something shorter, e.g.
```
trappy: skip empty array regex for common case
```
kudos for the big speedup are still reported in the change-log ;-)